src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useContext, useState } from 'react';
+import classNames from 'classnames';
+import { DispatchTodos, StateTodos } from '../TodosContext/TodosContext';
+import { Query } from '../../helpers/Query';
+
+interface Props {
+  setQuery: (value: Query) => void;
+}
+
+export const Footer: React.FC<Props> = ({ setQuery = () => { } }) => {
+  const [selected, setSelected] = useState('All');
```

Comment:
Add a generic types everywhere for state
```suggestion
  const [selected, setSelected] = useState<boolean>('All');
```